actions/check_branch_and_release_type/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Checks the branch and release type
+description: 'Enforces that pre-release builds are done on a release branch and release builds are done on the default branc'
+inputs:
+  release_type:
+    description: The type of version number to return. Must be one of [Snapshot, Patch, Minor or Major]
```

Comment:
Type `choice`?

Reply:
You don't specify a type for action inputs, so you can't set it to choice